client/my-sites/theme/themes-related-card/index.jsx

```diff
@@ -70,4 +74,8 @@ const ThemesRelatedCard = React.createClass( {
 	}
 } );
 
-export default ThemesRelatedCard;
+export default connect(
+	state => ( {
+		getDetailsUrl: theme => getThemeDetailsUrl( state, theme, getSelectedSiteId( state ) )
```

Comment:
I don't see anything wrong with adding the siteid here but in your comment you say it makes more contextual urls which I saw. I noticed that my site slug was added to the end of the URL but I'm just curious what effect that actually has. Are things enabled when I click around that weren't before?